src/CLI/ErrorCollector.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+
+namespace AxeWindowsScanner
+{
+    public class ErrorCollector : IErrorCollector
```

Comment:
It doesn't much matter because this is an exe and not a library, but is there any reason this class is public and not internal? Our typical pattern is to make classes like this internal.

Reply:
Changed. Options needs to remain public (for CommandLineParser), and VerbosityLevel needs to be remain public (since it's referenced by Options)